test/whitePaperModel/mining/borrowMiningTests.spec.ts

```diff
@@ -977,7 +984,7 @@ contract("borrowMiningTests", async (accounts) => {
                             console.log("balFINAfterUser1", balFINAfterUser1.toString());
 
                             expect(BN(balFINAfterUser2)).to.be.bignumber.equal(
-                                new BN("101000000000000000000000")
+                                new BN("151500000258173303249000")
```

Comment:
why this amount is changed ?

Reply:
Because previously it was only considering the borrow mining FIN for the borrowed DAI. After the change it is adding the deposit mining FIN as well.
This user has USDT and DAI deposits, on which it borrows DAI.